snorkel/labeling/lf/nlp.py

```diff
@@ -243,3 +224,48 @@ def __call__(self, f: Callable[..., int]) -> NLPLabelingFunction:
             disable=self.disable,
             memoize=self.memoize,
         )
+
+
+class nlp_labeling_function(base_nlp_labeling_function):
+    """Decorator to define an NLPLabelingFunction object from a function.
+
+    Parameters
+    ----------
+    name
+        Name of the LF
+    resources
+        Labeling resources passed in to ``f`` via ``kwargs``
+    pre
+        Preprocessors to run before SpacyPreprocessor is executed
+    fault_tolerant
+        Output -1 if LF execution fails?
+    text_field
+        Name of data point text field to input
+    doc_field
+        Name of data point field to output parsed document to
+    language
+        spaCy model to load
+        See https://spacy.io/usage/models#usage
+    disable
+        List of pipeline components to disable
+        See https://spacy.io/usage/processing-pipelines#disabling
+    memoize
+        Memoize preprocessor outputs?
+
+
+    Example
+    -------
+    >>> @nlp_labeling_function()
+    ... def has_person_mention(x):
+    ...     person_ents = [ent for ent in x.doc.ents if ent.label_ == "PERSON"]
+    ...     return 0 if len(person_ents) > 0 else -1
+    >>> has_person_mention
+    NLPLabelingFunction has_person_mention, Preprocessors: [SpacyPreprocessor...]
+
+    >>> from types import SimpleNamespace
+    >>> x = SimpleNamespace(text="The movie was good.")
+    >>> has_person_mention(x)
+    -1
```

Comment:
Do we not document inherited attributes?

Reply:
Nope, we should! I'll update

Reply:
oh sorry, the attributes for these decorators aren't accessible, so we skip those in doc